### PR TITLE
Geek Train Edit.

### PR DESCRIPTION
**Pull request by Adron**
Still needs the ticket links.

### CODE REVIEW

File: proposals/stuff-for-whatever [hunk 1]
```diff
@@ -0,0 +1 @@
+Stuff.
```
Comment (thoward) [inline proposals/stuff-for-whatever:1]: This looks like it wasn't supposed to be part of this PR.

File: index.html [hunk 2]
```diff
@@ -76,9 +76,9 @@
             <li><a href="#signup">Sign Up</a></li>
             <li><a href="#tickets">Tickets</a></li>
             <li><a href="#cfp">Call For Proposals</a></li>
+            <li><a href="#geektrain">Geek Train</a></li>
```
Comment (thoward) [inline index.html:79]: Looking good, but do we want this to be a top-level navigation item? Maybe instead we could make a "schedule" item here, and start building that out, with the GeekTrain as the first piece of that? Also, schedule should probably be a separate page, cause it's going to get large.

Comment (adron-orange) [inline index.html:79]: I gotta get back at this and am going to redo a few bits.

I'll actually bump the Geek Train to a separate landing page, that just links off of the schedule, etc. 

Needless to say will get updates posted shortly.

**Comment by Adron** [comment]
btw, I ended up PRing via @adron-orange for this, we can close this PR.

**Pull request closed without merging**